DependencyInjection/Configuration.php

```diff
@@ -41,24 +42,80 @@ public function getConfigTreeBuilder() : TreeBuilder
 
         $rootNode
             ->children()
-                ->scalarNode('dir_name')->defaultValue('%kernel.root_dir%/DoctrineMigrations')->cannotBeEmpty()->end()
-                ->scalarNode('namespace')->defaultValue('Application\Migrations')->cannotBeEmpty()->end()
-                ->scalarNode('table_name')->defaultValue('migration_versions')->cannotBeEmpty()->end()
-                ->scalarNode('column_name')->defaultValue('version')->end()
-                ->scalarNode('column_length')->defaultValue(14)->end()
-                ->scalarNode('executed_at_column_name')->defaultValue('executed_at')->end()
-                ->scalarNode('all_or_nothing')->defaultValue(false)->end()
                 ->scalarNode('name')->defaultValue('Application Migrations')->end()
-                ->scalarNode('custom_template')->defaultValue(null)->end()
-                ->scalarNode('organize_migrations')->defaultValue(false)
+                ->arrayNode('migrations_paths')
+
+                    ->info('A list of pairs namespace/path where to look for migrations.')
+                    ->requiresAtLeastOneElement()
+                    ->useAttributeAsKey('name')
+                    ->defaultValue(['%kernel.root_dir%/DoctrineMigrations' => 'Application\Migrations'])
+                    ->prototype('scalar')->end()
+                    ->validate()
+                        ->ifTrue(static function ($v) {
+                            return count($v) === 0;
+                        })
+                        ->thenInvalid('At least one migrations path must be specified.')
+                    ->end()
+                 ->end()
+
+                ->arrayNode('storage')
```

Comment:
Indentation looks off compared to the line above. This reminds me that I need to check out the multiline call sniff for doctrine/coding-standard.

Reply:
Note: indentation is still off.

Reply:
do not understand what is wrong with the indentation, can you clarify?

Reply:
It was the line above - I've left a separate comment on it